pkg/ovsdb/ovsdb.go

```diff
@@ -23,15 +23,34 @@ import (
 )
 
 // OVS driver state
-type OvsBridgeDriver struct {
+type OvsDriver struct {
 	// OVS client
 	ovsClient *libovsdb.OvsdbClient
+}
+
+type OvsBridgeDriver struct {
+	OvsDriver
 
 	// Name of the OVS bridge
 	OvsBridgeName string
 }
 
 // Create a new OVS driver with Unix socket
+func NewOvsDriver(ovsSocket string) (*OvsDriver, error) {
+	ovsDriver := new(OvsDriver)
+
+	ovsDB, err := libovsdb.ConnectWithUnixSocket(ovsSocket)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to ovsdb error: %v", err)
+	}
+
+	// Setup state
```

Comment:
I don't understand what do you want to say with this comment

Reply:
it's a leftover sorry